client/my-sites/stats/wordads/index.jsx

```diff
@@ -148,6 +150,11 @@ class WordAds extends Component {
 				/>
 				<PrivacyPolicyBanner />
 				<SidebarNavigation />
+				<FormattedHeader
+					className="wordads__section-header"
+					headerText={ translate( 'Stats and Insights' ) }
+					leftAlign
```

Comment:
I think this is a typo, but it's causing a weird jump when switching through Stats tabs. I proposed a fix in #37778. :)

<img width="950" alt="Screenshot 2019-11-20 at 17 05 42" src="https://user-images.githubusercontent.com/43215253/69260772-523f7580-0bb8-11ea-9526-f3c3c0b1b7c0.png">